bson/marshal_value_test.go

```diff
@@ -162,7 +78,20 @@ func TestMarshalValue(t *testing.T) {
 
 func compareMarshalValueResults(t *testing.T, tc marshalValueTestCase, gotType bsontype.Type, gotBytes []byte) {
 	t.Helper()
-	expectedValue := RawValue{Type: tc.expectedType, Value: tc.expectedBytes}
+	expectedValue := RawValue{Type: tc.bsontype, Value: tc.bytes}
 	gotValue := RawValue{Type: gotType, Value: gotBytes}
 	assert.Equal(t, expectedValue, gotValue, "value mismatch; expected %s, got %s", expectedValue, gotValue)
 }
+
+// benchmark covering GODRIVER-2779
+func BenchmarkSliceCodec_Marshal(b *testing.B) {
```

Comment:
What is the purpose of the "_" in the benchmark function name? Should it be removed?